projects/valtimo/dossier/src/lib/components/dossier-process-start-modal/dossier-process-start-modal.component.ts

```diff
@@ -75,6 +81,11 @@ export class DossierProcessStartModalComponent implements OnInit {
     this.processLinkId = null;
     this.formDefinition = null;
     this.formFlowInstanceId = null;
+    if (this.configService.config.featureToggles?.useStartEventNameAsStartFormTitle) {
```

Comment:
can you make this a property you set in the component so you dont need to duplicate it on line 119?